pkg/arvo/app/publish.hoon

```diff
@@ -245,7 +238,12 @@
     |=  [wir=wire sin=sign:agent:gall]
     ^-  (quip card _this)
     ?-    -.sin
-        %poke-ack   (on-agent:def wir sin)
+        %poke-ack
+      ?~  p.sin
+        (on-agent:def wir sin)
```

Comment:
While iirc the default behavior for no-error `%poke-ack`s is "do nothing", you explicitly want "do nothing" here, not whatever the default behavior is.

```suggestion
      ?~  p.sin  [~ this]
```